src/check-catalog/check-catalog.service.ts

```diff
@@ -2,9 +2,24 @@ import { EntityManager } from "typeorm";
 import { Inject, Injectable, OnApplicationBootstrap } from "@nestjs/common";
 import { VIEW_NAME } from "./check-catalog.constants";
 
+interface CheckItem {
+  code: string;
+  message: string;
+  plugins: string[];
+}
+
+interface FormattedValues {
+  stackPipeId?: string;
+  orisCode?: string;
+  systemID?: string;
+  unadjustedHrlyValue?: string;
+  sampleMethod?: string;
+  [key: string]: string | undefined;
+}
+
```

Comment:
This is the same as `Record<string, string>`:
```suggestion
type FormattedValues = Record<string, string>;
```